### PR TITLE
Add resolveOneCS to ActorSelection for Java CompletionStage 

### PR DESCRIPTION
**Pull request by richardimaoka**
Refs #21726

Does this look ok? I followed how [CircuitBreaker implemented callWithCircuitBreakerCS()](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala#L172L182).

JavaDoc looks like this.

image

### CODE REVIEW

**Comment by ktoso** [comment]
PLS BUILD

**Comment by ktoso** [comment]
Failure was a bit weird

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Comment by patriknw** [comment]
PLS BUILD

File: akka-actor/src/main/scala/akka/actor/ActorSelection.scala [hunk 1]
```diff
@@ -78,6 +82,19 @@ abstract class ActorSelection extends Serializable {
    */
   def resolveOne(timeout: FiniteDuration): Future[ActorRef] = resolveOne()(timeout)
 
+  /**
+   * Java API (8) for [[#resolveOne]]
```
Comment (johanandren) [inline akka-actor/src/main/scala/akka/actor/ActorSelection.scala]: No need for the `(8)`, we don't support earlier versions

**Review by johanandren — APPROVED** [review]

File: akka-actor/src/main/scala/akka/actor/ActorSelection.scala (hunk 1 above)
Comment (richard-imaoka) [inline akka-actor/src/main/scala/akka/actor/ActorSelection.scala]: Sure, removed (8) in the new commit.

**Comment by johanandren** [comment]
Thanks @richard-imaoka !

**Pull request merged**